linter.py

```diff
@@ -17,6 +17,30 @@ class Pyflakes(PythonLinter):
     defaults = {
         'selector': 'source.python'
     }
+    warning_msg_regex = r'''(?x)
+        \b(?:
+        used|
+        unused|
+        redefines|
+        shadowed|
+        (may\sbe)
+        )\b
+    '''
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.warning_msg_regex = re.compile(self.warning_msg_regex, 0)
+
+    def split_match(self, match):
+        # mark properly the type error message
```

Comment:
Remove as it's clear what we're doing here.  The comment is also not a sentence.  😁